pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -103,6 +106,10 @@ buildPythonPackage {
 
     pushd dist
 
+    for f in tensorflow-*+nv*.whl; do
+      mv "$f" "$(sed -E 's/(nv[0-9]+)\.0*([0-9]+)/\1.\2/' <<< "$f")"
```

Comment:
A comment explaining the pattern in English would be nice, maybe just an example `xxxx -> yyyy`

Reply:
Done